mathex/test-simple.c

```diff
@@ -18,17 +18,37 @@ static struct expr_func user_funcs[] = {
 
 int main()
 {
-    const char *s = "x = 40, add(2, x)";
-    struct expr_var_list vars = {0};
-    struct expr *e = expr_create(s, strlen(s), &vars, user_funcs);
-    if (!e) {
-        printf("Syntax error");
-        return 1;
-    }
+    const char s[][100] = {"1&&0", "~567", "1/3", "1/3*6", "1.3|0"};
+    // const char s[][100] = {"12773/ 31", "2+3", "4*5", "7-3", "-3", "99/2",
```

Comment:
Why did you leave the test case? Are they un-functioned?

Reply:
I forgot to remove the modification in mathex directory. That directory should be unused now since I've modified and moved `expression.c` and `expression.h` to the root directory.

Reply:
Good. I will remove directory `mathex` after this pull request is merged.